.github/workflows/ci.yml

```diff
@@ -114,6 +114,22 @@ jobs:
 
     steps:
       - uses: actions/checkout@v4
+      - name: Install Bowtie
```

Comment:
The goal is to not need any changes to the CI setup. I haven't reviewed the rest yet, it actually from a skim looked like it was probably correct or close to it though -- have you tried it without any of these CI changes?